clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystemCore.java

```diff
@@ -206,4 +206,43 @@ void setState(TachyonFile file, SetStateOptions options) throws IOException,
    */
   boolean unmount(TachyonURI tachyonPath, UnmountOptions options) throws IOException,
       TachyonException;
+
+  /**
+   * Sets the owner of a path.
+   *
+   * @param path the path of file or directory
+   * @param user the user to be set as owner
+   * @param recursive whether to set owner recursively under a directory
+   * @return true if set successfully, false otherwise
+   * @throws TachyonException if a Tachyon error occurs
+   * @throws IOException an I/O error occurs
+   */
+  boolean setOwner(TachyonURI path, String user, boolean recursive) throws TachyonException,
```

Comment:
for backwards compatibility, this function should be using the Options idiom and the recursive parameter should go in there